util/file_test.go

```diff
@@ -120,31 +121,27 @@ func TestJoinTerraformModulePath(t *testing.T) {
 	}
 }
 
-func TestPathContains(t *testing.T) {
+func TestFileManifest(t *testing.T) {
 	t.Parallel()
 
-	testCases := []struct {
-		description string
-		path        string
-		folderName  string
-		expected    bool
-	}{
-		{"both empty", "", "", true},
-		{"path empty, folder name not empty", "", "foo", false},
-		{"path non empty, folder name empty", "/foo", "", true},
-		{"path contains one part that matches folder name", "foo", "foo", true},
-		{"path contains one part that doesn't match folder name", "foo", "bar", false},
-		{"path contains multiple parts, one of which matches folder name", "/foo/bar/baz", "bar", true},
-		{"path contains multiple parts, none of which match folder name", "/foo/bar/baz", "will-not-match", false},
+	var testfiles = []string{"file1", "file2"}
```

Comment:
Thanks for adding tests! 👍 

However, I'd strongly recommend having tests for all the corner cases, as those are what'll kill us here:

1. Empty folder
1. Nested files/folders
1. Nested folders, some of which are empty
1. Etc